examples/src/components/GeoPushpinChartConfigurationPointsExample.jsx

```diff
@@ -0,0 +1,113 @@
+// (C) 2020 GoodData Corporation
+import React, { Component } from "react";
+import { GeoPushpinChart, Model } from "@gooddata/react-components";
+
+import "@gooddata/react-components/styles/css/main.css";
+
+import { projectId } from "../utils/fixtures";
+import { MAPBOX_TOKEN, cityCoordinatesUri, populationUri, cityNamesUri } from "../utils/fixturesGeoChart";
+
+const sizeMeasure = Model.measure(populationUri)
+    .format("#,##0.00")
+    .aggregation("sum")
+    .alias("Population");
+const locationAttribute = Model.attribute(cityCoordinatesUri).localIdentifier("location");
+const tooltipTextAttribute = Model.attribute(cityNamesUri).localIdentifier("tooltipText");
+
+export class GeoPushpinChartConfigurationPointsExample extends Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            config: {
+                tooltipText: tooltipTextAttribute,
+                mapboxToken: MAPBOX_TOKEN,
+                points: {
+                    groupNearbyPoints: true,
+                },
```

Comment:
could we only put `points` into `this.state`?

Reply:
Yes, I agree.
State must only contain mutateble values.

Reply:
fixed